benchmarks/dynamo/pr_time_benchmarks/benchmarks/aotdispatcher.py

```diff
@@ -0,0 +1,65 @@
+import sys
+
+from benchmark_base import BenchmarkBase
+
+import torch
+from torch.testing._internal.two_tensor import TwoTensor
+
+class Benchmark(BenchmarkBase):
+    def __init__(self, *, training, subclass):
+        self._training = training
+        self._subclass = subclass
+        self._device = "cpu"
```

Comment:
we always use cpu,  is there interesting code paths that we would miss for cuda tensors?

Reply:
Overall AOTDispatcher is pretty device generic. There are a few places where we can end up doing things for cpu vs cuda (e.g. a separate decomp in the `AutogradCPU` vs `AutogradCUDA` dispatch keys), but in general I think this is pretty rare / unlikely to affect compile times in a device-specific way.